content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -0,0 +1,51 @@
+---
+title: इंटरएक्टिव ट्यूटोरियल - एक ऐप की डेप्लॉयमेंट
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="row">
+            <div class="col-md-12">
+                <p>
+                    पॉड कुबेरनेट्स एप्लिकेशन की मूल निष्पादन इकाई है। प्रत्येक पॉड आपके क्लस्टर पर चल रहे कार्यभार के एक हिस्से का प्रतिनिधित्व करता है। <a href="/docs/concepts/workloads/pods/">पॉड्स के बारे में अधिक जानें</a>।
+                </p>
+            </div>
+        </div>
+
+        <br>
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                टर्मिनल का उपयोग करने के लिए, कृपया डेस्कटॉप या टैबलेट मोड का उपयोग करें
```

Comment:
```suggestion
                टर्मिनल का उपयोग करने के लिए, कृपया डेस्कटॉप या टैबलेट का उपयोग करें
```

In english, it's written as `To interact with the Terminal, please use the desktop/tablet version`, not sure if `version` plays any role in the sentence. I'll ask someone in Slack.

Reply:
I think `मोड` is necessary term, `डेस्कटॉप या टैबलेट का उपयोग करें` implies we need to use desktop or tablet, while what we wanted to say was we can use desktop mode on any device for katakoda

Reply:
Yeah, some apps can run on landscape mode of the phone as well.

Reply:
how about `टर्मिनल के साथ इंटरैक्ट  करने के लिए, कृपया डेस्कटॉप/टैबलेट संस्करण का उपयोग करें`, if we follow the word convention from the sheet [here ](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0)